sunpy/net/dataretriever/sources/tests/test_eve.py

```diff
@@ -126,3 +126,37 @@ def test_client_repr(LCClient):
     """
     output = str(LCClient)
     assert output[:50] == 'sunpy.net.dataretriever.sources.eve.EVEClient\n\nPro'
+
+
+def mock_querry_object(LCClient):
```

Comment:
```suggestion
def mock_query_object(LCClient):
```
and everywhere else.

Reply:
I did not notice.

Reply:
Oh, resolved it in all files.
I blindly copied spellings from already existing function here.

https://github.com/sunpy/sunpy/blob/db314114f0148d16610dda10f18ea0876942cd61/sunpy/net/dataretriever/sources/tests/test_goes_suvi.py#L65